FlorenceBML/BMLacq38.xml

```diff
@@ -34,6 +35,376 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <pb n="139r"/> <hi rend="rubric">ወደቂቀ፡</hi> ራሔል። ዮሴፍ። ወብንያሚ። እሉ፡ እሙንቱ፡ ፲ወ፪ነገድ። </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i9">
+                            <locus from="139r" to="143r"/>
+                            <title>List of the Kings of Ethiopia from Arwe to Ləbna Dəngəl</title>
+                            <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ ስሉስ፡ ፡ ፩አምላክ። </hi>ወእምዝ፡ ንጽሕፍ፡ ዓመታተ፡ ነገሥተ፡ አክሱም። ዘአርዌ፡ 
+                                <hi rend="rubric">፼</hi> ዓመት። ዘብእሲ፡ አንገቦ፡ ዘቀተሎ፡ ለአርዌ፡ <hi rend="rubric">፪፻</hi>። </incipit>
+                            <explicit xml:lang="gez">ውናግ፡ ሰገደ፡ ወ፡ ዘተሰምየ፡ ልብነ፡ ድን<pb n="143r"/>
+                                ግል፡ ፳ወ<supplied reason="lost">፫</supplied>፡ ዓመት። ወተጽሕፈት፡ ዛቲ፡ መጽሐፍ፡ በእደዊሁ፡ ለቀሲስ፡ 
+                                <persName ref="PRS11227ZaraLe" role="scribe" cert="low">ዘርአ፡ ልዑል።</persName> 
+                                በ፻፹፡ ወ<supplied reason="lost">፫</supplied>፡ ዓመተ፡ ምሕረት፡ በ<supplied reason="lost">ሉቃ</supplied>ስ፡ ወንጌላዊ። 
+                                ወዘሰረቃ፡ ወተአገላ፡ ውጉዘ፡ ይኩን፡ በአፋሆሙ፡ ለጴጥሮስ፡ ወጳውሎስ፡ ለዓለመ፡ ዓለም። </explicit>
+                            <note>The explicit contains a twofold date and a name of a scribe. Alas the date is not complete, because in several instances numerals
+                                have been intentionally erased, but can according to <bibl><ptr target="bm:Marrassini1987manoscrittietiopici"/>
+                                <citedRange unit="page">77</citedRange></bibl> be reconstructed to the year 23 of the reign of Ləbna Dəngəl and the year 183,
+                                the two dates that both give the year <date cert="medium" calendar="gregorian">1530</date> as date of production of the manuscript. 
+                                Whether the agency of the scribe <persName ref="PRS11227ZaraLe" role="scribe" cert="low">ዘርአ፡ ልዑል።</persName> 
+                                on the other hand relates to the entire manuscript or yet alone to the last three folios is not clear as well. Since the last bifolium 
+                                <locus target="#q20"/> is written with less care and with another pen, it may have been written by another hand and represent 
+                                a later addition.</note>
+                            <note>According to <bibl><ptr target="bm:Marrassini1987manoscrittietiopici"/><citedRange unit="page">77</citedRange></bibl>
+                                the list as far as the Kings of Aksum are concerned, corresponds to <title ref="LIT3855KingList"/> and <title ref="LIT3853KingList"/> 
+                                according to Conti Rossinis categorization 
+                                in <bibl><ptr target="bm:ContiRossini1909Listes"/><citedRange unit="page">268 and 285</citedRange></bibl>.</note>
+                        </msItem>
+                        <msItem xml:id="ms_i10">
```

Comment:
I agree that it is not a composite manuscript, the last bifolium is original, but in my opinion it is the protective quire which was originally blank and later filled, by a secondary hand, with a guest text. so I would say this `msItem `is an `additio`

@DenisNosnitsin1970 what do you think?

Reply:
`so group all chronological texts (3-8) in one msItem 3` 
Done and nested all in ms_i3

Reply:
Thank you for your comment! This meets with my own observations. I will change the definition of hands (h1+h2 > h1) and will note the deviation from Marrassini. The Śərʿata Qwərbān-text will be restored to ms-item again (and not additio).